"tsc watch" task broken for projects with references

<!-- Please search existing issues to avoid creating duplicates. -->
<!-- Also please test using the latest insiders build to make sure your issue has not already been fixed: https://code.visualstudio.com/insiders/ -->

<!-- Use Help > Report Issue to prefill these. -->
- VSCode Version: 1.30.2
- OS Version: macOS X 10.13.6 (17G4015)

Steps to Reproduce:

1. Open a package with `references` in the `tsconfig.json` file
2. Attempt to execute the `tsc : watch - tsconfig.json` build task
3. Behold the `tsc` error (pardon my french):
   ```
   > Executing task: tsc --watch -b /Repro/tsconfig.json <

   error TS6369: Option '--build' must be the first command line argument.
   Le processus du terminal s'est achevé avec le code de sortie 1

   Le terminal sera réutilisé par les tâches, appuyez sur une touche pour le fermer.
   ```

<!-- Launch with `code --disable-extensions` to check. -->
Does this issue occur when all extensions are disabled?: Yes

---

The issue appears to be caused by the `-b` being added at [`tasks.ts:221`](https://github.com/Microsoft/vscode/blob/e2cf8ebc5d6e827199124b911ff3e25ff38ae2ec/extensions/typescript-language-features/src/features/task.ts#L221) and `--watch` being prepended to that value at [`tasks.ts:200`](https://github.com/Microsoft/vscode/blob/e2cf8ebc5d6e827199124b911ff3e25ff38ae2ec/extensions/typescript-language-features/src/features/task.ts#L200), resulting in an invalid invocation of `tsc`. A possible solution might be to append `--watch` at the end instead of prepending.

--- a/extensions/typescript-language-features/src/features/task.ts
+++ b/extensions/typescript-language-features/src/features/task.ts
@@ -197,7 +197,7 @@ class TscTaskProvider implements vscode.TaskProvider {
 				project.workspaceFolder || vscode.TaskScope.Workspace,
 				localize('buildAndWatchTscLabel', 'watch - {0}', label),
 				'tsc',
-				new vscode.ShellExecution(command, ['--watch', ...args]),
+				new vscode.ShellExecution(command, [...args, '--watch']),
 				'$tsc-watch');
 			watchTask.group = vscode.TaskGroup.Build;
 			watchTask.isBackground = true;

